charts/sentry-apps/ci/default-values.yaml

```
@@ -0,0 +1,6 @@
+---
+ingress:
+  hostname: sentry.example.org
+
+sentry:
+  postgresqlPassword: changeme
```

We should ensure that `changeme` is never used.
```suggestion
  postgresqlPassword: 
```

---

this is `ci`

---

oh i see, sry